CHANGELOG.md

```diff
@@ -24,6 +24,11 @@
   for large files.  
   [Niil Öhlin](https://github.com/niilohlin)
 
+* Add new opt-in rule `hash_function` to warn against empty wrong hashValue
```

Comment:
This is incorrect and poorly worded. How about

> Add new `legacy_hashing` rule to encourage the use of Swift 4.2's new hashing interface.

Reply:
Changed the name. 

Also changed kind to `.idiomatic` to math the other legacy rules